spec/components/review_draft/apply_draft/view_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ReviewDraft::ApplyDraft::View do
+  alias_method :component, :page
+
+  describe "sections that appear for application draft trainee" do
+    before do
+      render_inline(described_class.new(trainee: trainee))
+    end
+
+    context "when the trainee is on the provider-led route", 'feature_routes.provider_led_postgrad': true do
+      let(:trainee) { create(:trainee, :provider_led_postgrad, :with_placement_assignment, :with_apply_application) }
+
+      it "renders the correct sections" do
```

Comment:
As above, are there sections that specifically _shouldn't_ render? If so, test here.

Reply:
yep, I've added some, thanks